channels/us.m3u

```diff
@@ -394,43 +394,43 @@ https://thegateway.app/BizAndYou/Biz_720p/playlist.m3u8
 https://adultswim-vodlive.cdn.turner.com/live/black-jesus/stream.m3u8
 #EXTINF:-1 tvg-id="BlackNewsChannel.us" tvg-name="Black News Channel" tvg-country="US" tvg-language="English" tvg-logo="https://image.xumo.com/v1/channels/channel/9999960/248x140.png?type=channelTile" group-title="News",Black News Channel (1080p)
 http://redbox-blacknewschannel-xumo.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="BlackNewsChannel.us" tvg-name="Black News Channel" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/3RUw9BW.png" group-title="",Black News Channel
+#EXTINF:-1 tvg-id="BlackNewsChannel.us" tvg-name="Black News Channel" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/3RUw9BW.png" group-title="News",Black News Channel
```

Comment:
When you see multiple sources of absolutely the same channel it's better to keep all tags identical (logo included) to avoid duplicating in generated playlists. Don't hesitste to change existing logos if you find that yours suits channel the best

Reply:
I was wondering do we keep the Pluto, Xumo, and Roku logo's? Can those be changed, so the viewer knows that's possibly the source?

Reply:
Pluto's got their images in their servers, not quite sure for XUMO and Roku.

If you find images that corresponds to the server's origin, please add them and change all the corresponding image links.

Reply:
When it comes to searching programs... well... there's channels here that aren't real channels, only streams that contains episodes of a TV show that was aired (All Weddings as an example is simply a channel made of all the episodes of "All Weddings" that were aired on TV.)

Reply:
Got it.